list-your-project/other-dashboards/fees-adapters.md

```diff
@@ -0,0 +1,308 @@
+# Building Fees Adapters
+
+This guide will help you create adapters for the [fees dashboard](https://defillama.com/fees) that track fees and revenue metrics for DeFi protocols.
+
+## Adapter Versions
+
+Before creating a fees adapter, you need to decide which version to use:
+
+- **V1 Adapters**: Run for fixed time periods (00:00 to 23:59 of a given day)
+- **V2 Adapters**: Accept arbitrary start and end timestamps as input
```

Comment:
lets make everyone do v2 adapters, v1 adapters are bad

i'd orient the entire guide around v2 adapters, then include a note saying that if its only possible to get data for the last 24h or only from 0:00 to 0:00, its possible to create an adapter for those cases too but its not recommended